### DISCUSSION TITLE
Can someone confirm if jquery-ui-1.13.1 is compatible with jquery-3.5.1 ?

### DISCUSSION THREAD

**Discussion by AnanyaBanerjee01** [jquery/jquery-ui#2066] [General]
Can someone confirm if jquery-ui-1.13.1 is compatible with jquery-3.5.1 ?

**Comment by mgol** [comment]
Yes, jQuery UI is compatible with all jQuery versions starting from 1.8 up until the latest stable one (currently 3.6).
